nucypher/blockchain/eth/agents.py

```diff
@@ -61,7 +61,8 @@ def get_balance(self, address: str=None) -> int:
 
 
 class NucypherTokenAgent(EthereumContractAgent):
-    _principal_contract_name = "NuCypherToken"
+    principal_contract_name = "NuCypherToken"
+    _upgradeable = False
```

Comment:
I really like the `_upgradeable` being here :-)

Reply:
:heavy_check_mark: